src/Morphir/IR/SDK.elm

```diff
@@ -12,6 +13,11 @@ import Morphir.IR.SDK.Result as Result
 import Morphir.IR.SDK.String as String
 
 
+packageName : Path
+packageName =
+    [ [ "morphir" ], [ "s", "d", "k" ] ]
+
```

Comment:
Just a stylistic thing. I think the below should be easier to read -

`Path.fromString "morphir.sdk"`

Reply:
Agreed. Update coming ...